dashboard/test/ui/features/project_gallery.feature

```diff
@@ -0,0 +1,45 @@
+# TODO (Erin B.) remove ?showFeatured=1 param from project gallery url during flag removal.
```

Comment:
Your other PR looks good, so you might actually want to just take care of this TODO in this PR.